### ISSUE TITLE
Document inherited methods?

### ISSUE THREAD

**Issue by zachcp** [machow/quartodoc#239]
Hi Quarto Devs, 

Is there currently a way to toggle documentation of inherited methods? I'm looking to document a subset of inherited methods from a parent class and these are not picked up automatically.

zcp

**Comment by machow** [comment]
Hey! This should be pretty quick to implement. I'll try to pick it up this week.

Any chance you could point me to the library you're documenting (if it's public)? I can use it to double check things are working, before merging in any changes.

**Comment by zachcp** [comment]
Thank you,  @machow!   I am trying to document a non-public API but you could imagine a class that inherits from PandasDataFrame and you would like to doument, say, the [plot ](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html) method.

**Comment by machow** [comment]
Released in v0.5.0 as the the `include_inherited` option!

E.g.

```yaml
quartodoc:
  package: quartodoc
  sections:
    - content
      - name: MdRenderer
        include_inherited: true
```

You can set default options across the whole project like so...

```yaml
quartodoc:
  package: quartodoc

  # sets include inherited everywhere by default ----
  options:
    include_inherited: true
  sections:

    # you can also set it directly in a specific section ---
    - options:
        include_inherited: true 
      content:
        - MdRenderer
```

**Comment by zachcp** [comment]
@machow  Thank you!  I'm just getting back from some time off. My initial test looks good.
